docs/openapi.yaml

```diff
@@ -95,7 +95,7 @@ paths:
         specified testnet address.
 
         The endpoint returns the transaction ID, which you can use to view the transaction in the
-        [Stacks Explorer](https://explorer.stacks.co/?chain=testnet). The tokens are delivered once the transaction has
+        [Stacks Explorer](https://explorer.hiro.so/?chain=testnet). The tokens are delivered once the transaction has
```

Comment:
although explorer.hiro.so is operations, we haven't fully migrated the explorer yet afaik 
cc: @CharlieC3